python/ray/autoscaler/_private/monitor.py

```diff
@@ -134,17 +134,20 @@ class Monitor:
 
     def __init__(self,
                  redis_address,
+                 gcs_address,
```

Comment:
What are the backwards compatibility implications here? It would be good to call them out in the PR description (e.g., do we have to upgrade external autoscaling operators to match the Ray version after this change?)

cc @AmeerHajAli

Reply:
I think as offline discussed, it shouldn't be an issue right now since it'll always use the matched version of autoscaler.